dash/include/dash/halo/Halo.h

```diff
@@ -55,18 +61,18 @@ class StencilPoint : public Dimensional<int16_t, NumDimensions> {
   constexpr StencilPoint(
     typename std::enable_if<sizeof...(Values) == NumDimensions - 1,
```

Comment:
I suggest moving this check into a `static_assert` because it's not SFINAE.

Reply:
it's SFINAE. i have no better solution in the moment to avoid an ambiguous constructor.

Reply:
Mhh, not sure I understand that. I see 2 ctor, both taking varargs, both with the same `std::enable_if`. Why not use a `static_assert` instead? What is the ambiguity?

Reply:
Ahh, I think I now better understand what is going on. The constructors might be ambiguous if `CoeffT` has the same type as the values?

Reply:
it's ambiguous also in case of different value types. It's kind of strange. My first attempt was to declare the constructors explicit, to avoid implicit conversions. Didn't worked. 

Btw. the enable if's are not the same. The value of size_of is the same, but the total number of  arguments differs.